src/Symfony/Component/VarDumper/Cloner/Data.php

```diff
@@ -14,7 +14,7 @@
 /**
  * @author Nicolas Grekas <p@tchwork.com>
  */
-class Data
+class Data implements \Serializable
```

Comment:
@wouterj can you show before/after content of serialized object? Looks like you pass all of the properties to serialize, does it some changes?